darwin/future/tests/data_objects/workflow/data/user.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Example file used for test - NB: all IDs are randomly generated/madeup, unrelated to production system IDs